source/Applications/Advanced/PoseConversions/PoseConversions.cs

```diff
@@ -0,0 +1,325 @@
+﻿/*
+This example shows how to perform Hand-Eye calibration.
+*/
+
+using System;
+using System.IO;
+using YamlDotNet.RepresentationModel;
+
+
+using MathNet.Numerics.LinearAlgebra;
+using MathNet.Numerics.LinearAlgebra.Double;
+
+class Program
+{
+    static void Main()
+    {
+        try
+        {
+            var zivid = new Zivid.NET.Application();
+
+            printHeader("This example shows conversions to/from Transformation Matrix");
+
+            // Read a transformation matrix
+            var transformationMatrixFile = Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData)
+                               + "/Zivid/RobotTransform.yaml";
+            Console.WriteLine("Getting PoseState:");
+            var transformationMatrix = readTransform(transformationMatrixFile);
+            var mathMatrix = zividToMathDotNet(transformationMatrix);
+            Console.WriteLine(matrixToString(mathMatrix));
+
+            // Extract rotation matrix and translation vector
+            var rotationMatrix = mathMatrix.SubMatrix(0, 3, 0, 3);
+            var translationVector = mathMatrix.SubMatrix(0, 3, 3, 1);
+
+            Console.WriteLine("RotationMatrix:\n" + matrixToString(rotationMatrix));
+            Console.WriteLine("TranslationVector:\n" + matrixToString(translationVector.Transpose()));
+
+            /*
+             * Convert from Rotation Matrix (Zivid) to other representations of orientation (Robot)
+             */
+            printHeader("Convert from Zivid (Rotation Matrix) to Robot");
+            var axisAngle = rotationMatrixToAngleAxis(rotationMatrix);
+            Console.WriteLine("AxisAngle:\n" + matrixToString(axisAngle.Item1.Transpose()) + ", " + String.Format(" {0:G4} ", axisAngle.Item2));
+            var rotationVector = axisAngle.Item1 * axisAngle.Item2;
```

Comment:
Create a function called rotationMatrixToRotationVector, see: https://github.com/zivid/zivid-cpp-samples/blob/d840629c50b18d6428ae6749774497f310980dbf/source/Applications/Advanced/HandEyeCalibration/PoseConversions/PoseConversions.cpp#L181-L185
In that function, you can call rotationMatrixToAngleAxis and do this.
One of the points of these samples is for customers to chose which functions they need and then copy paste them and use them. So we want to have a function for each conversion, like in C++ code sample.

Reply:
Done